js.mk

```diff
@@ -9,25 +9,17 @@ JS_FILES ?= media/js
 
 NODE_MODULES ?= ./node_modules
 JS_SENTINAL ?= $(NODE_MODULES)/sentinal
-JSHINT ?= $(NODE_MODULES)/jshint/bin/jshint
```

Comment:
Instead of editing this file manually, you can run `make update_makefiles` in this directory to download the new version of this file from ccnmtl/makefiles. That way, the version at the top is updated as well.

Reply:
Will do. I started this work before the ccnmtldjango stuff was updated.